src/__tests__/posthog-core.js

```diff
@@ -599,7 +599,7 @@ describe('init()', () => {
         delete window.rrwebRecord
         window.rrwebRecord = 'is possible'
         given.subject()
-        expect(given.lib.__loaded_recorder_version).toMatch(new RegExp(`^1\.?`)) // start with 1.?.?
```

Comment:
This was doing a string escape, not a regex escape, which I don't think was what was intended